Content.Server/Resist/ResistLockerSystem.cs

```diff
@@ -0,0 +1,117 @@
+using Content.Shared.Movement;
+using Robust.Shared.GameObjects;
+using Content.Server.Storage.Components;
+using Content.Server.DoAfter;
+using Content.Server.Lock;
+using Robust.Shared.IoC;
+using Robust.Shared.Player;
+using Robust.Shared.Containers;
+using Content.Server.Popups;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Resist;
+
+public class ResistLockerSystem : EntitySystem
+{
+    [Dependency] private readonly DoAfterSystem _doAfterSystem = default!;
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly LockSystem _lockSystem = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ResistLockerComponent, RelayMovementEntityEvent>(OnRelayMovement);
+        SubscribeLocalEvent<ResistLockerComponent, ResistDoAfterComplete>(OnDoAfterComplete);
+        SubscribeLocalEvent<ResistLockerComponent, ResistDoAfterCancelled>(OnDoAfterCancelled);
+        SubscribeLocalEvent<ResistLockerComponent, EntRemovedFromContainerMessage>(OnRemovedFromContainer);
+    }
+
+    private void OnRelayMovement(EntityUid uid, ResistLockerComponent component, RelayMovementEntityEvent args)
+    {
+        if (!TryComp(uid, out EntityStorageComponent? storageComponent))
```

Comment:
Now that this isn't a resolve, move this check to after the isResisting check so that the simpler check runs first.